samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -96,8 +96,8 @@ static const char sock_send_usage_str[] =
 	"                            The number is between 0001..9999 and it can roll over.\n"
 	"                            Must be used with -d and -e options and without\n"
 	"                            -l and -x options.\n"
-	"  -B, --blocking, [int]     Blocking (1) or non-blocking (0) mode. This is only\n"
-	"                            valid when -l is given. Default value is 1.\n"
+	"  -B, --blocking, [int]     Blocking (1) or non-blocking (0) mode.\n"
+	"                            Default value is 1.\n"
```

Comment:
I guess this doesn't work with `-e`. Unless we change it, we should mentioned that this is not supported with `-e` option.

Reply:
I don't see why `-B` shouldn't work with `-e`. Blocking or non-blocking shouldn't affect multiple send?

Reply:
`set_sock_blocking_mode()` is not used for `-e` if I'm not mistaken.
Earlier `set_sock_blocking_mode()` was only used with `-l`, and now it's done also with `-d` but not with `-d` and `-e`.

Reply:
Right. Do we want to keep this restriction, or do we add support for using `-B` with `-e`?

Reply:
I documented this for now.

Reply:
Ok. I don't see a problem for adding `-B 1 -e` support too if you want.